src/main/scala/diplomaticobjectmodel/logicaltree/RocketLogicalTreeNode.scala

```diff
@@ -83,7 +84,7 @@ class RocketLogicalTreeNode(
     val omBusError = components.collectFirst { case x: OMBusError => x }
 
     Seq(OMRocketCore(
-      isa = OMISA.rocketISA(tile, XLen),
+      isa = OMISA.rocketISA(tile, XLen, PgLevels),
```

Comment:
why is this in this PR? Maybe we should be making this against the 3.3 bump to see more clearly what is added by the linter and what is added by the chisel bump?